api/v1/inputs.go

```diff
@@ -78,6 +78,11 @@ type Ref struct {
 // A reference to a resource kind/group.
 type ResourceRef struct {
 	// +required
-	Kind  string `json:"kind"`
 	Group string `json:"group,omitempty"`
+
+	// +required
+	Version string `json:"version,omitempty"`
```

Comment:
I changed my mind on this - I think we should specify a version instead of always returning latest